.github/workflows/pr.yaml

```diff
@@ -0,0 +1,14 @@
+name: Run CI
+on: [push, pull_request]
+jobs:
+
+  build:
+    name: Run CI
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out the code
+      uses: actions/checkout@v1
```

Comment:
I think we want to use checkout@v2? @nrb bumped it for velero core (https://github.com/vmware-tanzu/velero/blob/master/.github/workflows/pr.yml) but I don't recall exactly why, maybe something with tags.

Reply:
I think we are using `v2` in the csi plugin repo as well. Switched to `v2`